Mathlib/CategoryTheory/Monoidal/Tor.lean

```diff
@@ -22,6 +22,8 @@ Possibly it's best to axiomatize delta functors, and obtain a unique characteris
 -/
 
 
+assert_not_exists ModuleCat.abelian
```

Comment:
I think this (renaming) change is in another PR as well. But I don't care strongly about separating these PRs.

Reply:
I'll just leave it in: it has been wrong for a while now, so I don't think that it is especially important to track this!